site/_docs/reference.md

```diff
@@ -1861,6 +1861,28 @@ Not implemented:
 * REGR_SLOPE(numeric1, numeric2)
 * REGR_SXY(numeric1, numeric2)
 
+#### Ordered-Set Aggregate Functions
+
+Syntax:
+
+{% highlight sql %}
+aggregateCall:
+      agg '(' value ')'
+      [ WITHIN GROUP '(' ORDER BY orderItem ')' ]
```

Comment:
I think i have to remove the square brackets since the WITHIN GROUP clause is mandatory.

Reply:
+1